build/RELEASING.md

```diff
@@ -5,6 +5,16 @@ Only for Maintainers.
  1. Decide the tag name (version name) to be released. e.g. 1.4.0-beta.1,
     1.0.0-rc9.7 etc.
 
+    Notes:
+
+       * Instrumentation packages are core unstable packages always depend on
```

Comment:
@vishweshbankwar when can these packages move to contrib repo?

Reply:
We should be able to move them: https://github.com/open-telemetry/opentelemetry-dotnet/issues/5526. Looking for help.